H5P.BranchedVideo/branched-video.js

```diff
@@ -930,17 +930,38 @@ H5P.BranchedVideo = (function ($) {
       volumeSlider.max = 100;
       volumeSlider.value = 100;
       volumeSlider.style.display = 'none';
+      var lastSliderValue = 100;
       volumeButton.onmouseenter = function(){
         volumeSlider.style.display = 'block';
         var currentVid = getBranch(self.currentPlayingSlug).getVideoHTML();
         var volume = currentVid.volume * 100;
         volumeSlider.value = volume;
       };
       volumeDiv.onmouseleave = function(){volumeSlider.style.display = 'none';};
+      volumeButton.onmousedown = function() {
+        if (volumeSlider.value == 0) {
+          //UNMUTE
+          volumeSlider.value = lastSliderValue;
+          volumeButton.className = 'tapestry-volume-button';
+        } else {
+          //MUTE
+          volumeSlider.value = 0;
+          volumeButton.className = 'tapestry-mute-button';
```

Comment:
(same as above) Is this line necessary? (since you are already changing the class on line 960)